src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECPoint.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public sealed class ECPoint
+    {
+        public byte[] x;
+        public byte[] y;
```

Comment:
Part of an API review, but I'd expect these to be properties with upper-case names.